packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -59,7 +59,10 @@ export const generateTestDataForSlug = async (
     }
   }
 
-  const featuresConfig = getTemplatesConfig(templateModuleCollection);
+  const featuresConfig = getTemplatesConfig(
+    templateModuleCollection,
+    undefined
```

Comment:
What does undefined represent?

Reply:
It means no redirectTemplateCollection. I'll change to use ? notation so we don't need to pass undefined